lib/power_action_utils.pm

```diff
@@ -81,12 +81,19 @@ sub reboot_x11 {
     my ($self) = @_;
     wait_still_screen;
     if (check_var('DESKTOP', 'gnome')) {
-        if (is_tumbleweed) {
+        # Get gnome version
+        x11_start_program('xterm -geometry 150x45+5+5', target_match => 'xterm');
+        become_root;
```

Comment:
Why elevate to root? a regular user can very well read the rpm database

Reply:
It's not to read the rpm database, I had this [issue](http://angmar.suse.de/tests/2625#step/reboot_gnome/4)

Reply:
oh.. that's … unexpected; as we write to /dev/ttyS0 all the time.

And this likely only happens in your test case as you shortcut too much. you skip the consoletest_setup, that assigns the proper permissions

Reply:
But it also happens [in this opensuse test](http://angmar.suse.de/tests/2632#step/reboot_and_install/7) for instance

Reply:
I see; that test also just boots from the disk image and never setups the serial console  permissions (conisoletest_setup); which leads to this situation. Which is why http://angmar.suse.de/tests/2632/modules/ssh_key_check/steps/1/src also needs the become_root step in there.

Reply:
I understand, all I mean is that I should not remove the become_root line, given that it will fail in that specific scenario.

Reply:
still same thing, you need `consoletest_setup` right after `boot_to_desktop`

Reply:
I 'm all for removing the `become_root` line.
It is just that we should add `consoletest_setup` after `boot_to_desktop` to [gnome+do_not_import_ssh_keys](https://openqa.opensuse.org/tests/1480600#step/reboot_and_install/1), [external_iso]
(https://openqa.opensuse.org/tests/1480703#step/reboot_and_install/1) and [gnome+import_ssh_keys]
(https://openqa.opensuse.org/tests/1480694#step/reboot_and_install/1) so that they don't fail, unless I am misunderstanding something

Reply:
yep, that's how it should be

Reply:
So, for the current PR, I will leave `become_root` as it is, so that I don't break upstream jobs.
When `consoletest_setup` starts being scheduled before `reboot_and_install` in those jobs, I will submit a new PR and remove `become_root`.
Is this ok?

Reply:
@ggkioulis Sounds good